mstranslator.py

```diff
@@ -89,143 +94,109 @@ def make_response(self, resp):
 
         return data
 
-    def _translate(self, action, text_params, lang_from, lang_to, contenttype, category):
+    def _translate(self, action, body, lang_from, lang_to, contenttype, category, includeAlignment = False):
         if not lang_to:
             raise ValueError('lang_to parameter is required')
         if contenttype not in ('text/plain', 'text/html'):
             raise ValueError('Invalid contenttype value')
 
         params = {
             'to': lang_to,
-            'contentType': contenttype,
             'category': category,
+            'includeAlignment': 'true' if includeAlignment else 'false'
         }
+
         if lang_from:
             params['from'] = lang_from
-        params.update(text_params)
 
-        return self.make_request(action, params)
+        return self.make_request(action, params, body)
 
     def translate(self, text, lang_from=None, lang_to=None,
                   contenttype='text/plain', category='general'):
-        params = {
-            'text': text,
-        }
-        return self._translate('Translate', params, lang_from, lang_to,
+        body = [{
+            'text' : text
+        }]
+        response = self._translate('translate', body, lang_from, lang_to,
                                contenttype, category)
+        if 'error' in response:
+            raise ArgumentOutOfRangeException(response['error']['message'])
+        else:
+            return response[0]['translations'][0]['text']
 
     def translate_array(self, texts=[], lang_from=None, lang_to=None,
                         contenttype='text/plain', category='general'):
-        params = {
-            'texts': json.dumps(texts),
-        }
-        return self._translate('TranslateArray', params, lang_from, lang_to,
+        body = [
```

Comment:
As I can see from [docs](https://docs.microsoft.com/en-us/azure/cognitive-services/translator/migrate-to-v3), there is no difference between `Translate` and `TranslateArray` methods now. Shouldn't we combine methods with `_array` suffix into one?

Reply:
```suggestion
        def _array(self, texts=[], lang_from=None, lang_to=None, contenttype='text/plain', category='general', include_alignment=False):
                json = [
                    {'Text' : text} for text in texts
                ]
                return self._translate('translate', json, lang_from, lang_to, contenttype, category, include_alignment)
        
        def translate_array(self, texts=[], lang_from=None, lang_to=None,
                        contenttype='text/plain', category='general'):
                return self._array(self, texts, lang_from, lang_to, contenttype, category)

        def translate_array2(self, texts=[], lang_from=None, lang_to=None,
                        contenttype='text/plain', category='general'):
                return self._array(self, texts, lang_from, lang_to, contenttype, category, include_alignment=True)
```
Do yo mean something like this?